mailgreen/app/main.py

```
@@ -25,9 +27,14 @@
     keyword_router,
     trash_router,
     star_router,
-    carbon_router
+    carbon_router,
```

Ruff (0.11.9)

`carbon_router` may be undefined, or defined from star imports

(F405)